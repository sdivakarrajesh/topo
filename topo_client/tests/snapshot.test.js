import { render } from '@testing-library/react';
import QuarterlyRevenueChart from '../app/components/quarterly_trends'; // Import the helper function if it's separated

describe('QuarterlyRevenueChart Snapshot', () => {
    const mockData = [
        { quarter: 'Q1', revenue: 2000000 },
        { quarter: 'Q2', revenue: 2500000 },
    ];

    it('should match the snapshot', () => {
        const { asFragment } = render(<QuarterlyRevenueChart data={mockData} />);
        expect(asFragment()).toMatchSnapshot(); // This will store a snapshot of the rendered component
    });
});
