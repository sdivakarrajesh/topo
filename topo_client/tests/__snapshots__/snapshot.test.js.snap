// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuarterlyRevenueChart Snapshot should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="recharts-responsive-container"
    style="width: 100%; height: 300px; min-width: 0;"
  />
</DocumentFragment>
`;
